IP/src/main/java/TaskList.java

```diff
@@ -0,0 +1,96 @@
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskList {
+    private Storage storage;
+    private Ui ui;
+    private static List<Task> listOfTasks = new ArrayList<>();
+    private final String STORAGEFILEPATH = "Meerkat.txt";
+
+    public TaskList() {
+        this.storage = new Storage();
+        this.ui = new Ui();
+    }
+
+    public String taskListToString(List<Task> taskList) {
+        StringBuilder s = new StringBuilder();
```

Comment:
Would suggest to use stringBuilder variable name rather than just 's' to avoid ambiguity with mostly Scanner object.

Reply:
Noted, will implement the changes! Thank you :)